app/pb_kits/playbook/pb_typeahead/_typeahead.scss

```diff
@@ -6,14 +9,16 @@
       position: absolute;
       width: min-content;
       bottom: 0.6em;
-      left: 1em;
-      visibility: hidden;
+      right: 1em;
+      opacity: 0;
+      color: $text_lt_light;
+      transition: opacity .15s ease-in-out;
     }
   }
 
   [class^=pb_text_input_kit] {
     input {
-      padding-left: $space_xl !important;
+      padding-right: $space_xl !important;
```

Comment:
Any way to avoid this `!important` here? Stronger selector?

Reply:
I can try and let you know